packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/shouldContinueProfiling..st

```diff
@@ -0,0 +1,3 @@
+accessing
+shouldContinueProfiling: aBoolean
```

Comment:
@linusha: Actually, after thinking about it some time, I think `isCurrentlyProfiling` would be perfectly fine as a name for this instance variable. Maybe even better than `shouldContinueProfiling`… Sorry!

Reply:
I think it comes down to the discussion we had about the "withNextStep" thing. Since we agreed, that 1ns is sillly, I will rename the variable in my next commit.